pytorch_lightning/loggers/base.py

```diff
@@ -126,7 +129,10 @@ def _aggregate_metrics(
         return agg_step, agg_mets
 
     def _reduce_agg_metrics(self):
-        """Aggregate accumulated metrics."""
+        """Aggregate accumulated metrics.
+
+        .. deprecated:: v1.6     This method is deprecated in v1.6 and will be removed in v1.8.
```

Comment:
```suggestion
        .. deprecated:: v1.6
            This method is deprecated in v1.6 and will be removed in v1.8.
```

Reply:
Strange... your suggestion is exactly what I wrote but it looks like pre-commit changed it here https://github.com/PyTorchLightning/pytorch-lightning/pull/11832/commits/1b7a1bd05ac2fdecc7c1d765c54e3056b23a28b3

If I commit your suggestion, pre-commit will probably change it back again. I'll play around with the docstring and find something that pre-commit likes.

Reply:
See https://github.com/PyCQA/docformatter/issues/78 for the workaround